src/main/java/teammates/common/util/Assumption.java

```diff
@@ -8,9 +8,7 @@
  * an unchecked {@link AssertionError} will be thrown at runtime.
  *
  * <p>Normally, we use Java assertion to do runtime checking, but GAE does not support the assertions.
- * The methods of this file is adapted from org.junit.Assert v4.10.
- *
- * @see org.junit.Assert
+ * The methods of this file are adapted from org.junit.jupiter.api.Assertions v5.3.1.
```

Comment:
My comment is not related to this PR: The reason we introduced the Assumptions class was because earlier versions of GAE did not support the native assert statement. May want to check if we still need this class.